test/e2e/snaps/test-snap-siginsights.spec.js

```diff
@@ -253,57 +239,6 @@ describe('Test Snap Signature Insights', function () {
         });
 
         // TEST FIVE: eth_sign
-        // switch to metamask window
-        await driver.switchToWindowWithTitle(
-          WINDOW_TITLES.ExtensionInFullScreenView,
-        );
-
-        // click on the global action menu
-        await driver.waitForSelector(
-          '[data-testid="account-options-menu-button"]',
-        );
-        await driver.clickElement(
-          '[data-testid="account-options-menu-button"]',
-        );
-
-        // try to click on the settings item
-        await driver.clickElement({
-          text: 'Settings',
-          tag: 'div',
-        });
-
-        // try to click on the advanced item
-        await driver.clickElement({
-          text: 'Advanced',
-          tag: 'div',
-        });
-
-        // scroll to and click on Eth_sign requests toggle
-        const ethSignToggle1 = await driver.findElement('.eth-sign-toggle');
-        await driver.scrollToElement(ethSignToggle1);
-        await driver.clickElement('.eth-sign-toggle');
-
-        // wait for and click checkbox on modal and click continue
-        await driver.waitForSelector('#eth-sign-checkbox');
-        await driver.clickElement('#eth-sign-checkbox');
-        await driver.clickElement({
-          text: 'Continue',
-          tag: 'button',
-        });
-
-        // fill in annoyance field and click enable
-        await driver.waitForSelector('#enter-eth-sign-text');
-        await driver.fill(
-          '#enter-eth-sign-text',
-          'I only sign what I understand',
-        );
-        await driver.clickElement({
-          text: 'Enable',
-          tag: 'button',
-        });
-
-        // switch back to test-dapp window
-        await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
```

Comment:
we don't need all the steps for enabling eth sign in settings, as it's done with fixtures now